src/fare-contracts/components/FareContractFromTo.tsx

```diff
@@ -57,7 +57,13 @@ export const FareContractFromTo = (props: FareContractFromToProps) => {
     if (hasFareContract(props)) {
       const travelRight = props.fc.travelRights[0];
       if (isNormalTravelRight(travelRight)) {
-        return travelRight.direction;
+        if (!!travelRight.direction) {
+          // A travelRight between quays (e.g. for boat)
+          return travelRight.direction;
+        } else if (travelRight.tariffZoneRefs?.length ?? 0 > 1) {
+          // A travelRight between several zones (e.g. for bus)
+          return TravelRightDirection.Both;
+        }
       }
     } else if (hasRecentFareContract(props)) {
       return props.rfc.direction;
```

Comment:
Does recent fare contract always have direction? I would think the same issue with checking if multiple zones is present here.

Coming to think of it since zones doesn't have direction I think we can just send the `TravelRightDirection` directly to the `HarborsFromTo`. While `ZonesFromTo` is hardcoded internally to use `Both`.

Reply:
Clever thinking, but for RecentFareContracts it is only ever shown [if direction is defined](https://github.com/AtB-AS/mittatb-app/blob/master/src/recent-fare-contracts/RecentFareContract.tsx#L165) so the issue is masked there. 

But that might change, so I'll take a look at your suggestion!